src/MultiplicationTable.ts

```diff
@@ -1,5 +1,32 @@
 export class MultiplicationTable {
-  public render(start: number, end: number): string{
-    return '1*1=1'
+  public render(start: number, end: number): string {
+    return !this.isValidInputs(start, end) ? "" : this.generateMultable(start, end);
+  }
+  isValidInputs(start: number, end: number): boolean {
+    return (!this.isValidRange(start, end) || !this.isInputsInteger(start, end))?false:true;
+  }
+  isValidRange(start: number, end: number): boolean {
+    return this.isWithinRange(start) && this.isWithinRange(end) && (start <= end);
+  }
+  isWithinRange(input:number){
+    return input<=10&&input>=0;
+  }
+  isInputsInteger(start: number, end: number): boolean {
+    return Number.isInteger(start) && Number.isInteger(end);
+  }
+  generateMultable(start: number, end: number): string {
```

Comment:
'generateMultiplicationTable' is clearer than 'generateMultable' as a method name.